pkgs/by-name/ba/basalt-monado/package.nix

```diff
@@ -0,0 +1,112 @@
+{ stdenv,
+  lib,
+  boost,
+  eigen,
+  tbb_2021_8,
```

Comment:
With versioned packages like these, you should take `tbb` as an input and then manually `callPackage` this derivation in `top-level/all-packages.nix` with the corresponding version

This is currently one of the shortcomings of [RFC-140](https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md)